vm/chef/cookbooks/mariadb/files/mariadb-setup

```diff
@@ -0,0 +1,90 @@
+#!/bin/bash -eu
+#
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+source /opt/c2d/c2d-utils || exit 1
+
+set -o pipefail
+
+readonly hostname="$(hostname)"
+readonly mysql_root_password="$(get_attribute_value "MYSQL_ROOT_PASSWORD")"
+readonly mysql_replication_password="$(get_attribute_value "MYSQL_REPLICATION_PASSWORD")"
+
+readonly internal_ip="$(get_internal_ip)"
+
+# Get the list of MariaDB nodes
+readonly cluster_hostname_list="$(get_attribute_value "ENV_MARIADB_NODE_HOSTNAMES" | tr '|' ' ')"
+echo "MariaDB instances: ${cluster_hostname_list}"
+
+# Bootstrap the cluster
+server_id=1
+
+# Emits a timestamped message to the logs
+emit_msg() {
+  echo "$(date +"%I:%M:%S")" "${@}"
+}
+readonly -f emit_msg
+
+# Loop through nodes in the cluster, set the first one, node 0, as the primary
+for node in ${cluster_hostname_list}; do
+  echo $node
```

Comment:
It will not work with for node in "${cluster_hostname_list}";

Example: 
#!/bin/bash
var="aaa bbb ccc"

for i in ${var};
  do echo $i;
done

for i in "${var}";
  do echo $i;
done

Output:
aaa
bbb
ccc
aaa bbb ccc